Dockerfile

```diff
@@ -0,0 +1,112 @@
+# SPDX-License-Identifier: GPL-2.0-or-later
+#
+# Copyright 2020 (C) Olliver Schinagl <oliver@schinagl.nl>
+
+# hadolint ignore=DL3007  latest is the latest stable for alpine
+FROM registry.hub.docker.com/library/alpine:latest AS builder
+
+LABEL Maintainer="Olliver Schinagl <oliver@schinagl.nl>"
+
+WORKDIR /src
+
+COPY . /src/
+
+# hadolint ignore=DL3008  We want the latest stable versions
+RUN apk add --no-cache \
+        bsd-compat-headers \
+        bzip2-dev \
+        cmake \
+        curl-dev \
+        file \
+        fts-dev \
+        g++ \
+        git \
+        json-c-dev \
+        libmilter-dev \
+        libtool \
+        libxml2-dev \
+        linux-headers \
+        make \
+        ncurses-dev \
+        openssl-dev \
+        pcre2-dev \
+        python3 \
+        zlib-dev \
+    && \
+    mkdir -p "./build" && \
+    cd "./build" && \
+    cmake .. \
+          -DAPP_CONFIG_DIRECTORY="/etc/clamav" \
+          -DCMAKE_BUILD_TYPE"=release" \
+          -DCMAKE_INSTALL_FULL_BINDIR="/usr/bin" \
+          -DCMAKE_INSTALL_FULL_INCLUDEDIR="/usr/include" \
+          -DCMAKE_INSTALL_FULL_LIBDIR="/usr/lib" \
+          -DCMAKE_INSTALL_FULL_SBINDIR="/usr/sbin" \
+          -DCMAKE_INSTALL_LIBDIR="/usr/lib" \
+          -DCMAKE_INSTALL_PREFIX="/usr" \
+          -DENABLE_CLAMONACC=OFF \
+          -DENABLE_DOCS=OFF \
+          -DENABLE_EXAMPLES=OFF \
+          -DENABLE_JSON_SHARED=ON \
+          -DENABLE_MILTER=ON \
+          -DENABLE_STATIC_LIB=OFF && \
+    make DESTDIR="/clamav" -j$(($(nproc) - 1)) install && \
+    rm -r \
+       "/clamav/usr/include" \
+       "/clamav/usr/lib/pkgconfig/" \
+    && \
```

Comment:
Please remove this block.
a. I'd like to keep the include directory, in case people want to use the headers. 
b. The the pkgconfig directory didn't exist in my testing and trying to remove it failed the build.

Reply:
odd. But not sure why you want to have the include files, it's a container no?

Ok one use case I can imagine: FROM clamav:latest; RUN do stuff that needs headers.

The headers are cheap to keep, I don't mind. I wonder if we can track later how many people would actually use this though ...

Reply:
The use case you thought of is basically what I was thinking.  But I suppose anyone who wants to build something around clamav could also just copy our Dockerfile as a baseline.  Your plan to remove these makes sense then. Sorry!

Augh `lib64` got me!